internal/services/redhatopenshift/redhat_openshift_cluster_resource.go

```diff
@@ -135,6 +136,19 @@ func (r RedHatOpenShiftCluster) Arguments() map[string]*pluginsdk.Schema {
 						Sensitive:    true,
 						ValidateFunc: validation.StringIsNotEmpty,
 					},
+					"cluster_resource_group_name": {
```

Comment:
Having looked through other examples in the provider, this might want to be called `managed_resource_group_name` instead 
```suggestion
					"managed_resource_group_name": {
```